src/compositor/seat/pointer.h

```diff
@@ -27,6 +27,16 @@ struct wlc_view;
 struct wlc_pointer_origin {
    double x, y;
 };
+// struct for keeping the focused surface
+// used because we cannot otherwise determine the position
+// of a subsurface without going all the way to the toplevel view
+// scale_w and scale_h are used only when the focused surface is the surface of the view
```

Comment:
scale_w and scale_h isn't here anymore. wlc_focused_surface should make it already quite clear what this does IMO